pkg/platform/kube/controller/nucliofunction.go

```diff
@@ -114,6 +115,20 @@ func (fo *functionOperator) CreateOrUpdate(ctx context.Context, object runtime.O
 		return nil
 	}
 
+	skipFunctionDeploy := false
+	if skipFunctionBuildStr, ok := function.Annotations[functionconfig.FunctionAnnotationSkipDeploy]; ok {
```

Comment:
I thought when we talked, we decided not to change the function spec, and use annotations instead.
Otherwise there's no need to use the annotations at all, no?

Reply:
I added a function that checks the annotation so in the flow it looks like `function.SkipDeploy()`

Reply:
Yeah, exactly what I meant.